core/src/main/java/org/apache/iceberg/rest/responses/LoadTableResponse.java

```diff
@@ -94,6 +96,11 @@ public Builder withTableMetadata(TableMetadata tableMetadata) {
       return this;
     }
 
+    public Builder withMetadataLocation(String metaLocation) {
+      this.metadataLocation = metaLocation;
+      return this;
+    }
```

Comment:
This isn't used presently, but I've found myself having to rebuild the `TableMetadata` in a few places to get `metadataLocation` to be non-null. 

Given that the accessor in `LoadTableResponse` _also_ calls the builder and adds `metadataLocation` to it, this could be useful.